global_settings/models.py

```diff
@@ -28,7 +28,7 @@ class Meta:
 
 @register_setting(icon='cogs')
 class CloudfrontDistribution(BaseSetting):
-    distribution_id = models.CharField(max_length=255)
+    distribution_id = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
should this just be `id` (or `aws_id`) so you have `distribution.id` instead of `distribution.distribution_id`? (this is minor, feel free to ignore)